lib/logstash/outputs/datadog_metrics.rb

```diff
@@ -9,8 +10,7 @@
 
 # Default `queue_size` and `timeframe` are low in order to provide near realtime alerting.
 # If you do not use Datadog for alerting, consider raising these thresholds.
-
-class LogStash::Outputs::DatadogMetrics < LogStash::Outputs::Base
+module LogStash module Outputs class DatadogMetrics < LogStash::Outputs::Base
```

Comment:
I'm wondering if we like this change in one line? or having this in multiple lines? I tend to like this changes in multiple lines as simplify somehow readability for me. What do you think?